_posts/2022-04-13-stepping-out-of-the-knowledge-zone.md

```diff
@@ -0,0 +1,177 @@
+---
+layout: epic
+title: Stepping Out Of The Knowledge Zone
+subtitle: How I went out to face my fears
+date: 2022-04-13
+categories: [teams, velocity, integrity, cypress, forque, horizon]
+author: [kaja]
+comment_id: 715
```

Comment:
Is this the right comment_id? Looks like it points to [a thread](https://github.com/artsy/artsy.github.io/issues/715) for an earlier blog post.